content/get-started.ts

```diff
@@ -104,23 +106,28 @@ const getStartedContent: GetStartedContent = {
         '**Price: $30 - $70**\n\n**Estimated Turnaround: 2 weeks**\n\nIf the WATCH printout is not good enough, go to your local police or sheriff\'s office for an official “record review/challenge” fingerprint card.\n\nNext, write WSP a letter asking them to send an official copy of your complete CHRI. Include the fingerprint card and a money order. A copy of your CHRI will be returned to the address on the fingerprint card. Send everything to:\n\nWashington State Patrol Identification and Background Section\n\nPO Box 42633\n\nOlympia WA 98504-2633',
     },
   ],
-  rejectionReasons: [
+  step3Substeps: [
     {
-      id: 'rejection1',
+      id: 'Step 3.1',
```

Comment:
`ids` should be snake-case since they are often used to create html ids for elements.